### ISSUE TITLE
Dead links in documentation

### ISSUE THREAD

**Issue by jmynes** [KingJigglypuff/project-plus-ex#1]
# Tl;dr
I was making a local backup of whatever is linked in the various documentation, just for posterity, and have come across a few dead links

## Document 1
[BrawlEx Guide for P+Ex](https://docs.google.com/document/d/1ZoL_qDcwUpUXg82cKaUp-6D_AcfpFctoW6GXFY74_0k/edit?tab=t.0) contains a few dead links:

### Dead Link 1:
```
The Porting Tools Collection (provided by Codes/TheGag96) will make your life much easier if you need to edit SFX or GFX IDs.
```

The Discord CDN link for the file mentioned above is dead:
https://cdn.discordapp.com/attachments/278000207906734081/940810713360904242/Porting_Tools.zip

### Dead Link 2:
```
For vBrawl Effect.pac IDs, refer to the OpenSA Wiki Effect.pac IDs page.
```
Dead wiki link:
http://opensa.dantarion.com/wiki/Effect.pac_IDs

### Dead Link 3:
```
For vBrawl Trace IDs, refer to the OpenSA Wiki Traces page.
```
Dead wiki link:
http://opensa.dantarion.com/wiki/Traces

---

## Document 2
[Internal ID References](https://docs.google.com/document/d/1mQaRwM4dh9g1owel6_ZFNSVwmllbO1pcC7IaYtmJBzM/edit?tab=t.0) also has one dead link:
```
(Written by KingJigglypuff)
This document was made using the image below as a reference point (made by Nebulon, updated by Sylv) and serves to list the Config IDs used by the vBrawl cast, special characters, PM clones, P+ clones, and P+Ex clones.
```
https://cdn.discordapp.com/attachments/356144764766388224/1158535968652021831/image.png

---

## Document 3
[P+ Ex Guide](https://docs.google.com/document/d/1c5MxVhxe_YsPt48_j8Z7-IsbzRXAsOeoz-ynucUM4C8/edit?tab=t.0) also has a dead link:
```
Additionally, if you’re using Action Overrides, be sure to update any commands and/or external Goto/Sub Routine Offsets (IE: ones that reference coding outside the character’s file) to use what P+ uses. You can look over the P+ Fighter.pac Dump to know what to change.
```
https://cdn.discordapp.com/attachments/637721724502278149/798644169651388446/P_Fighter.pac_Extracted.txt

**Comment by jmynes** [comment]
I wasn't sure where else to report this, e.g. Discord, etc, hope this catches the right set of eyes

**Comment by KingJigglypuff** [comment]
Thank you for reporting this. I've since updated all the expired Discord links to external permanent links.
